### ISSUE TITLE
Textures not copying while converting to non-binary gltf

### ISSUE THREAD

**Issue by vsantosu** [pissang/claygl#53]
While doing [PR](https://github.com/pissang/claygl/pull/52) I found out that when textures are matched and the URI is changed in the non-binary GLTF, the images are not moved to the output directory.

Example: fbx2gltf.py model.fbx -o ./subdir/model.gltf

Here, all images in the local directory will stay in the local directory and not copy to ./subdir along the gltf file, which breaks the URI of the generated gltf file.

**Comment by pissang** [comment]
Good point, copying image is necessary in your case. Currently in the scripts only fixes the image URI. Will implement it this weekend

**Comment by nickkraakman** [comment]
This problem does not only occur with the `-o` flag, but also with the `-b` flag, or when using a .zip file (with an FBX and textures inside) as input. 

Created an issue here: https://github.com/pissang/clay-viewer/issues/35
